### DISCUSSION TITLE
Will the changes to Reddit's API affect libreddit?

### DISCUSSION THREAD

**Discussion by cameronj86** [spikecodes/libreddit#789] [General]
https://libreddit.oxymagnesium.com/r/reddit/comments/12qwagm/an_update_regarding_reddits_api/

https://libreddit.domain.glass/r/apolloapp/comments/12ram0f/had_a_few_calls_with_reddit_today_about_the/jgup8uy/

**Comment by sigaloid** [comment]
This is being discussed in #785, check there for some of our thoughts

**Comment by cameronj86** [comment]
Whoops, thanks for the heads up
